.ci/Jenkinsfile

```diff
@@ -88,6 +90,45 @@ pipeline {
         }
       }
     }
+    /**
+     Publish Docker images.
+     */
+    stage('Publish Docker image'){
+      when {
+        allOf {
+          anyOf {
```

Comment:
this filter which branches create a `docker.elastic.co/package-registry/distribution` Docker image